src/Microsoft.VisualStudio.Services.Agent/TraceSetting.cs

```diff
@@ -5,19 +5,23 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Runtime.Serialization;
+using Agent.Sdk.Knob;
+using Microsoft.VisualStudio.Services.Agent.Util;
 
 namespace Microsoft.VisualStudio.Services.Agent
 {
     [DataContract]
     public class TraceSetting
     {
+        private static UtilKnobValueContext _knobContext = new UtilKnobValueContext();
```

Comment:
Now that I am seeing this more, should UtilKnobValueContext be a singleton?

Reply:
Good idea! Done